src/l500/l500-private.h

```diff
@@ -258,14 +258,14 @@ namespace librealsense
 
         };
 
-        static std::map<uint16_t, std::string> device_to_fw_min_version = {
-            { L500_RECOVERY_PID, "1.4.1.0"},
-            { L535_RECOVERY_PID, "1.4.1.0"},
-            { L500_USB2_RECOVERY_PID_OLD, "1.4.1.0"},
-            { L500_PID, "1.4.1.0"},
-            { L515_PID_PRE_PRQ, "1.4.1.0"},
-            { L515_PID, "1.4.1.0"},
-            { L535_PID, "1.4.1.0"}
+        static std::map<uint16_t, std::pair<std::string, std::string>> device_to_fw_min_max_version = {
```

Comment:
Why not a pair of `firmware_version`... or even `char const *`?

Reply:
We can consider `firmware_version` as enhancement separately, I'm not sure whether it improves readability.